src/main/resources/application.conf

```diff
@@ -62,6 +62,14 @@ etc-client {
     }
   }
 
+  mining {
+    tx-pool-size = 1000
+    ommers-pool-size = 30
+    block-cashe-size = 30
+    coin-base = "0011223344556677889900112233445566778899"
```

Comment:
minor detail: it's usually written as one word: `coinbase`
of course it's not a blocker for this PR